src/Module.Server/Modes/Conquest/CrpgConquestServer.cs

```diff
@@ -45,6 +46,20 @@ public override MissionLobbyComponent.MultiplayerGameType GetMissionType()
 
     public MBReadOnlyList<FlagCapturePoint> AllCapturePoints { get; private set; } = new(new List<FlagCapturePoint>());
 
+    public override void OnPeerChangedTeam(NetworkCommunicator peer, Team oldTeam, Team newTeam)
+    {
+        var crpgPeer = peer.GetComponent<CrpgPeer>();
+        if (crpgPeer?.User == null)
+        {
+            return;
+        }
+
+        if (oldTeam == null && newTeam.Side == BattleSideEnum.Attacker)
+        {
+            crpgPeer.RewardMultiplier = Math.Max(crpgPeer.RewardMultiplier, 3);
+        }
+    }

```

Comment:
Why is that needed?

Reply:
starting the attackers at 3x multi ensured the equality of XP gained. Also thematically i thought this made sense, as defenders start to rack up multi instantly and get good rewards, while attackers always remained at 1x multi.
In my docs there is also an alternative of starting at 2x multi if that is too much.

Reply:
oooh ok that's the reason. Could you comment your choice in the code, I thought that was about when a guy gets rebalanced in another team.

Reply:
In my opinion it's generally harder to attack than to defend and helping the attackers get some rewards at the start made the rewards more fair compared to defenders

Reply:
Sure I'm good with that. Could you add a comment in the code please.

Reply:
Done